src/quo2/components/info/page_nav.cljs

```diff
@@ -0,0 +1,219 @@
+           mid-section-right-icon mid-section-description mid-section-description-color mid-section-description-icon mid-section-description-user-icon
+           mid-section-main-text-icon-color left-section-icon left-section-icon-color
+           left-section-icon-bg-color right-section-icons]
+    :or   {align-mid                         false
+           page-nav-color                    :transparent
+           horizontal-description?           false
+           one-icon-align-left?              false
+           page-nav-background-uri           ""
+           mid-section-type                  :text-with-description
+           mid-section-icon                  :wallet
+           mid-section-main-text             "# general"
+           mid-section-left-icon             :wallet
+           mid-section-right-icon            :wallet
+           mid-section-description           "Description"
+           mid-section-description-color     "black"
+           mid-section-description-icon      :placeholder20
+           mid-section-description-user-icon "https://i.picsum.photos/id/810/200/300.jpg?hmac=HgwlXd-OaLOAqhGyCiZDUb_75EgUI4u0GtS7nfgxd8s"
+           mid-section-main-text-icon-color  "nil"
+           left-section-icon                 :peach20
+           left-section-icon-color           "none"
+           left-section-icon-bg-color        colors/neutral-30
+           right-section-icons               [{:bg       colors/primary-40
+                                               :no-color true
+                                               :icon     :main-icons/placeholder20}]}}]
+  (let [{:keys [height width]}      (dimensions/window)
+        put-middle-section-on-left? (or (true? align-mid)
+                                        (> (count right-section-icons) 1))
+        mid-section-props {:mid-section-type                 mid-section-type
+                           :horizontal-description?          horizontal-description?
+                           :mid-section-main-text            mid-section-main-text
+                           :one-icon-align-left?             one-icon-align-left?
+                           :mid-section-right-icon           mid-section-right-icon
+                           :mid-section-icon                 mid-section-icon
+                           :mid-section-main-text-icon-color mid-section-main-text-icon-color
+                           :mid-section-left-icon            mid-section-left-icon}]
+    [rn/view {:style (cond->
+                      {:display            :flex
+                       :flex-direction     :row
+                       :width              width
+                       :height             (* 0.075 height)
+                       :align-items        :center
+                       :padding-horizontal 8
+                       :justify-content    :space-between}
+                       (string/blank? page-nav-background-uri) (assoc :background-color page-nav-color)
```

Comment:
most likely it won't be empty string, so it can be just  `page-nav-background-uri (assoc :background-color page-nav-color)`

Reply:
Done